client/my-sites/design-preview/index.js

```diff
@@ -98,7 +103,9 @@ export default function designPreview( WebPreview ) {
 				return;
 			}
 			debug( 'loading preview with customizations', this.props.customizations );
-			this.props.fetchPreviewMarkup( this.props.selectedSiteId, this.props.previewUrl, this.props.customizations );
+			// No leading slash allowed in pathname but we need a trailing slash
+			const path = this.props.previewUrl ? url.parse( this.props.previewUrl ).pathname.replace( /^\//, '' ) + '/' : null;
```

Comment:
Maybe this should be done in `fetchPreviewMarkup` or the actual API call instead of here? It's very much a specific-to-this-endpoint quirk and should probably be hidden behind some abstraction.